config/locales/sv.yml

```diff
@@ -478,6 +478,10 @@ sv:
       confirm_are_you_sure: *are_you_sure_confirm
       add_address: 'Lägg till adress'
       click_to_edit: 'klicka för att redigera'
+      edit_branding_status: Redigera Varumärkesstatus
+      branding_license: Varumärkesrättigheter
```

Comment:
H-märkets användningsrättigheter och policy
*** 
(?) To link to a document describing the Policy and Terms of Use of the H-branding image?

Reply:
I wonder if we could use a shorter message here, and add a tooltip with a more complete explanation.  This longer message messes up my carefully designed (hmmm ...) page:

<img width="509" alt="screen shot 2017-11-25 at 7 51 25 am" src="https://user-images.githubusercontent.com/9968213/33230665-ac08170a-d1b5-11e7-9b91-cb5665452735.png">

Later, we could add the link to the policy-and-terms-of-use page to the label or the tooltip.

Reply:
I didnt realize it was a label, I thought it was a link to the ToS ^_- Or would be. 
I think this one is moot as if they don't check the box they can't pay? Is it needed?

If needed then just "Accepterar policy" maybe?